docs/source/devguide/contributing.rst

```diff
@@ -0,0 +1,114 @@
+  contributors can be involved in decision making.
+
+Overview
+--------
+
+OpenMC uses a liberal contribution model for project governance. Anyone involved
+in development in a non-trivial capacity is given an opportunity to influence
+the direction of the project. Project decisions are made through a
+consensus-seeking process rather than by voting.
+
+Terminology
+-----------
+
+- A *Contributor* is any individual creating or commenting on an issue or pull
+  request.
+- A *Committer* is a subset of contributors who are authorized to review and
+  merge pull requests.
+- The *TC* (Technical Committee) is a group of committers who have the authority
+  to make decisions on behalf of the project team in order to resolve disputes.
+- The *Project Lead* is a single individual who has the authority to make a final
+  decision when the TC is unable to reach consensus.
+
+Contribution Process
+--------------------
+
+Any change to the OpenMC repository must be made through a pull request (PR).
+This applies to all changes to documentation, code, binary files, etc. Even long
+term committers and TC members must use pull requests.
+
+No pull request may be merged without being reviewed.
+
+For non-trivial contributions, pull requests should not be merged for at least
+36 hours to ensure that contributors in other timezones have time to review.
+Consideration should also be given to weekends and other holiday periods to
+ensure active committers have reasonable time to become involved in the
+discussion and review process if they wish.
+
+The default for each contribution is that it is accepted once no committer has
+an objection. During review committers may also request that a specific
+contributor who is most versed in a particular area review the PR before it can
+be merged. Once all issues brought by committers are addressed it can be merged
+by any committer.
+
+In the case of an objection being raised in a pull request by another committer,
+all involved committers should seek to arrive at a consensus by way of
+addressing concerns being expressed through discussion, compromise on the
+proposed change, or withdrawal of the proposed change.
+
+If objections to a PR are made and committers cannot reach a consensus on how to
+proceed, the decision is escalated to the TC. TC members should regularly
+discuss pending contributions in order to find a resolution. It is expected that
+only a small minority of issues be brought to the TC for resolution and that
+discussion and compromise among committers be the default resolution mechanism.
+
+Becoming a Committer
+--------------------
+
+All contributors who make a non-trivial contribution will be added as a
+committer in a timely manner. Committers are expected to follow this policy.
+
+TC Process
+----------
+
+The TC uses a "consensus seeking" process for issues that are escalated to the
+TC. The group tries to find a resolution that has no objections among TC
```

Comment:
The first sentence feels a little redundant with the word `TC`.  Consider rewording to something like

> Any issues brought to the TC will be addressed among the committee with a consensus-seeking process.